### PR TITLE
Added some modals for SubOrgForm

### PR DESCRIPTION
**Pull request by nightwarriorftw**
# Description
Just a small PR to check created modal for SubOrgForm are correct or not

Partially Fixes:  # 3 

## Type of change
Created some modals for SubOrgForm

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have not added a commit to any .db files as part of my pull request
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by nightwarriorftw** [comment]
@botanicvelious Please review

File: gsoc/models.py [hunk 1]
```diff
@@ -20,24 +20,100 @@ def __str__(self):
...
+
+
+class SubOrgAdminEmail(models.Model):# SubOrg Admin Email
+    suborg_admin_email = models.EmailField(name='suborg_admin_email')
+
+    def __str__(self):
+        return str(self.suborg_admin_email)
+
+
+class MentorEmail(models.Model):# Mentor Email
```
Comment (botanicvelious) [inline gsoc/models.py]: There will be more then one of these fyi, you could do it this way but you need some way to reference it back then.

File: gsoc/models.py [hunk 2]
```diff
@@ -20,24 +20,100 @@ def __str__(self):
...
+        return self.suborg
+
+
+
+
 class UserProfile(models.Model):
     user = models.OneToOneField(User, on_delete=models.CASCADE)
     gsoc_year = models.ManyToManyField(GsocYear, blank=True)
-    suborg_full_name = models.ManyToManyField(SubOrg, blank=True)
+    suborg_full_name = models.ManyToManyField(SubOrg, blanl=False)
```
Comment (botanicvelious) [inline gsoc/models.py]: typo?

File: gsoc/settings.py [hunk 3]
```diff
@@ -160,7 +160,12 @@
     'sortedm2m',
     'taggit',
     'gsoc',
-    'debug_toolbar'
+    'debug_toolbar',
+    'absolute',
+    'aldryn_forms',
+    'aldryn_forms.contrib.email_notifications',
+    'captcha',
+    'emailit',
```
Comment (botanicvelious) [inline gsoc/settings.py:169]: you using any of these?

File: gsoc/models.py [hunk 4]
```diff
@@ -54,7 +130,85 @@ def gsoc_year(self):
...
+            
 auth.models.User.add_to_class('suborg_full_name', suborg_full_name)
 auth.models.User.add_to_class('gsoc_year', gsoc_year)
+auth.models.User.add_to_class('reason', reason)
+auth.models.User.add_to_class('num_of_mentor', num_of_mentor)
+auth.models.User.add_to_class('suborg_admin_email', suborg_admin_email)
+auth.models.User.add_to_class('get_involve', get_involve)
+auth.models.User.add_to_class('keep_involve', keep_involve)
+auth.models.User.add_to_class('accepted', accepted)
+auth.models.User.add_to_class('suborg', suborg)
```
Comment (botanicvelious) [inline gsoc/models.py]: Why would you add these to the user?

File: gsoc/models.py [hunk 5]
```diff
@@ -54,7 +130,85 @@ def gsoc_year(self):
...
+        return None
+
+
+
+def suborg(self):
+    try:
+        user_profile = UserProfile.objects.get(user=self.id)
+        return user_profile.suborg.all()
+    except SubOrg.DoesNotExist:
+        return None
```
Comment (botanicvelious) [inline gsoc/models.py]: what is all of this for?

**Review by botanicvelious — CHANGES_REQUESTED** [review]

**Pull request closed without merging**